ext/pcre/config0.m4

```diff
@@ -35,7 +35,7 @@ PHP_ARG_WITH(pcre-jit,,[  --with-pcre-jit         Enable PCRE JIT functionality
       if test -x "$PCRE2_CONF"; then
         AC_MSG_CHECKING(for PCRE2 10.30 or greater)
         PCRE2_VER=`$PCRE2_CONF --version`
-        if test "`echo $PCRE2_VER | sed 's,\.,,g'`" -lt 1030; then
+        if test "`echo $PCRE2_VER | $SED 's/\([[0-9]]*\)\.\([[0-9]]*\)\(.\)*/\1\2/g'`" -lt "1030"; then
```

Comment:
For sed noobs, why is it necessary to write `[[0-9]]` instead of `[0-9]`?

Reply:
Hello, these are duplicated because of the m4 where `[` and `]` are quotes https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Active-Characters.html#Active-Characters

So the `[[` becomes `[`.

Reply:
Thanks! I though this was some weird sed thing...